src/components/HomeLandingPage/EventCard.tsx

```diff
@@ -8,9 +8,14 @@ import CardOptions from './CardOptions';
 
 interface EventCardProps {
   date: Date;
+  eventId: string;
   eventTitle: string;
   address: string;
   handleClick: () => void;
+  handleArchiveEvent: () => void;
+  handleUnarchiveEvent: () => void;
+  handleDeleteEvent: () => void;
```

Comment:
minor refactoring nit - since these callbacks will only be used within this component, is it possible to set up the mutations here and just pass in the `event` object to this component? i see that `HomeLandingPage` does have a `refetch` call that might not work in this context, so if this isn't possible that's fine too

Reply:
hmm yeah I could move these mutations to the EventCard component, but then I'd need to pass in the `event` object and the `refetch()` function. I'm thinking I should keep the mutation operations out of the EventCard component as much as possible, to make it as 'pure' of a component as possible. Any thoughts?